<launch>
  <node pkg="robot_position_estimate" name="robot_position_estimate_node" type="robot_position_estimate_node" output="screen">
  <param name="fu" type="double" value="376.629954"/>
  <param name="fv" type="double" value="494.151786"/>
  
  <param name="listen_h_flag" type="bool" value="false"/>
  <param name="loc_h" type="double" value="1.0"/>
  </node>
</launch>


